lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -400,12 +388,8 @@ sub get_current_job {
 
     return $self->reply->not_found if (!defined $self->param('testid'));
 
-    my $job = $self->schema->resultset("Jobs")->search(
-        {
-            id => $self->param('testid')
-        },
-        {prefetch => qw(jobs_assets)})->first;
-    return $job;
+    return $self->schema->resultset("Jobs")->search({id => $self->param('testid')}, {prefetch => qw(jobs_assets)})
+      ->first;
```

Comment:
```suggestion
    return $self->schema->resultset('Jobs')->find($self->param('testid'}, {prefetch => qw(jobs_assets)});
```

Reply:
should have tried a multi-line regex :D Fixed and found many more instances